src/test/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/InputIndentationInvalidArrayInitIndent.java

```diff
@@ -7,7 +7,7 @@
  * basicOffset = 4                                                            //indent:1 exp:1
  * braceAdjustment = 0                                                        //indent:1 exp:1
  * caseIndent = 4                                                             //indent:1 exp:1
- * forceStrictCondition = false                                               //indent:1 exp:1
+ * forceStrictCondition =                                                     //indent:1 exp:1
```

Comment:
please put exact value here if it is default, you can do `(default) false`.
https://checkstyle.sourceforge.io/checks/misc/indentation.html#Properties

same for other Input